app/assets/stylesheets/search.scss

```diff
@@ -26,4 +26,8 @@
     top: 18px;
   }
 
-}
\ No newline at end of file
+}
+
+//table {
```

Comment:
Maybe remove it?

Reply:
I suspect in post amigos Joe will notice the table is wider than it should be, which is when I can uncomment this